seed/xylem/gluster.py

```diff
@@ -144,17 +148,23 @@ def call_createdirs(self, args):
     def call_createvolume(self, args):
 
         name = args['name']
+        vol = yield self.getVolume(name)
 
-        vols = yield self.getVolumes()
+        if vol is None:
+            # The volume doesn't exist, let's create it.
+            yield self.createVolume(name)
+            vols = yield self.getVolumes()
```

Comment:
Mostly just wanting something to still use `getVolumes()` so it doesn't suffer bitrot.

It's possible that we'll want to expose a "list volumes" API or something in the future, and keeping this around in a known-good state will make that easier.